### PR TITLE
Add a set of scripts to collect data from PowerVS

### PR DESCRIPTION
**Pull request by rpsene**
Signed-off-by: Rafael Peria de Sene <rpsene@br.ibm.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/18#" title="Author self-approved">rpsene</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag**
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](?repo=ocp-power-automation%2Finfra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/18#issuecomment-701876585" title="Approved">mkumatag</a>*, *<a href="https://github.com/ocp-power-automation/infra/pull/18#" title="Author self-approved">rpsene</a>*

The full list of commands accepted by this bot can be found [here](?repo=ocp-power-automation%2Finfra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by mkumatag** [comment]
/hold
for others to review

File: scripts/powervs-data-collector/README.md
Comment (bpradipt) [inline scripts/powervs-data-collector/README.md:3]: nit: "PowerVS instance" => "PowerVS service instance"

File: scripts/powervs-data-collector/README.md
Comment (bpradipt) [inline scripts/powervs-data-collector/README.md:23]: nit: "yout" => "your"

**Comment by bpradipt** [comment]
Thanks @rpsene .. Overall looks good to me.. Few minor comments to be addressed

**Comment by ltccci** [comment]
New changes are detected. LGTM label has been removed.

**Comment by rpsene** [comment]
@bpradipt changes applied.

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/18#pullrequestreview-500355198" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/infra/pull/18#issuecomment-701876585" title="Approved">mkumatag</a>*, *<a href="https://github.com/ocp-power-automation/infra/pull/18#" title="Author self-approved">rpsene</a>*

The full list of commands accepted by this bot can be found [here](?repo=ocp-power-automation%2Finfra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)~~ [bpradipt,mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
